### PR TITLE
AN-Frontend challenge card update

### PR DESCRIPTION
**Pull request by alinix1**
JIRA Ticket link: [PC-xx](https://makeitmvp.atlassian.net/browse/PC-xx)

## Type of change

- [ ] Bug fix
- [ ] New feature
- [x] Refactoring

## Description

- Changes made? Added a border to the challenge card, changed font color of difficulty level, changed pixels to rem in the SCSS file, changed button styling to match Figma better, added dynamic display of images, and tweaked grid gap for ChallengePage component. 
- Reason for changes? Try and match Figma designs as closely as possible. 
- Behavior before? The design was not matching in every detail for the challenge card.
- Behavior after? The design more closely mimics the Figma file.
- How did you test? Locally in the browser. 

## Checklist

- [ ] I have performed a self-review of my code.
- [x] I have manually tested the new feature and found it to be working as expected.
- [x] All commits in this PR are related to the issue.
- [x] Base branch for this PR is set to `develop`.
- [x] JIRA ticket # and my initials are noted in this PR's name (format example: #57-EB Add Pull Request template)

**Reminder**: every PR needs at least 1 peer approval before merging to the `develop` branch. Once a main ticket has been completed, along with all subtasks, associated code can be merged to `main`.

### CODE REVIEW

**Comment by alinix1** [comment]
In Firebase I wonder if we need to change challenge data from difficulty level "Intermediate" to "Medium" for button display to match Figma design?

**Comment by alinix1** [comment]
I can also update ChallengeCard.scss to utilize _variables.scss in styles folder; i.e., $MVP-black

File: src/components/ChallengeCard/ChallengeCard.jsx [hunk 1]
```diff
@@ -14,34 +22,72 @@ const ChallengeCard = ({ challenge }) => {
...
         return "#FFD22F";
       case "hard":
-        return "#FF5733";
+        return "#FF7070";
       default:
         return "#37AFFF";
     }
   };
 
+  const getRandomBannerImage = (difficulty) => {
```
Comment (TimWallaceDev) [inline src/components/ChallengeCard/ChallengeCard.jsx:31]: This is great. It will be nice to have a variety of images

**Review by TimWallaceDev — APPROVED** [review]
This look good to merge

File: src/components/ChallengeCard/ChallengeCard.scss [hunk 2]
```diff
@@ -1,117 +1,119 @@
...
-            padding: 10px 0;
+            border-radius: 0.25rem;
+            border-top: 3px solid var(--colors-neutrals-800, #131313);
+            border-right: 6px solid var(--colors-neutrals-800, #131313);
+            border-bottom: 6px solid var(--colors-neutrals-800, #131313);
+            border-left: 3px solid var(--colors-neutrals-800, #131313);
+            width: 22.8125rem;
+            height: 3rem;
+            padding: 0.625rem 3rem;
+            margin-top: 2rem;
```
Comment (jagadeepavula) [inline src/components/ChallengeCard/ChallengeCard.scss:98]: added margin top to match the figma design

**Review by jagadeepavula — APPROVED** [review]

**Pull request merged**